vignettes/ggplot2-specs.Rmd

```diff
@@ -234,6 +232,9 @@ ggplot(df, aes(1, x)) +
   geom_text(aes(label = fontface, fontface = fontface))
 ```
 
+### Font size
+
+The `size` of text is measured in mm. 1 pt = 0.35 mm.
```

Comment:
Maybe add one additional sentence explaining what this means in practice? Something like: "1 pt = 0.35 mm, and ggplot2 provides this conversion factor in the variable `.pt`, which is defined as the number of points corresponding to 1 mm. Therefore, if you want to draw text in 12pt font, set `size = 12/.pt`."